packages/bbob-parser/src/lexer.ts

```diff
@@ -209,11 +216,12 @@ export function createLexer(buffer: string, options: LexerOptions = {}): LexerTo
     const isClosingTag = substr[0] === SLASH;
 
     if (isNoAttrsInTag || isClosingTag) {
+      const startPos = chars.getPos() - 1;
       const name = chars.grabWhile((char) => char !== closeTag);
 
       chars.skip(); // skip closeTag
 
-      emitToken(TYPE_TAG, name);
+      emitToken(TYPE_TAG, name, startPos, startPos + name.length + 2);
```

Comment:
need to extract to var `startPos + name.length + 2` also need define `2` as constant